docs/language/ql-handbook/expressions.rst

```diff
@@ -393,16 +393,73 @@ aggregation in a simpler form:
 Monotonic aggregates
 ====================
 
-In addition to the standard aggregates, QL also supports monotonic aggregates.
-These are a slightly different way of computing aggregates which have some advantages.
-For example, you can use monotonic aggregates :ref:`recursively <recursion>`. 
-You can't do this with normal aggregates.
+In addition to standard aggregates, QL also supports monotonic aggregates.
+Monotonic aggregates differ from standard aggregates in the way that they deal with the
+values generated by the ``<expression>`` part of the formula:
+
+- Standard aggregates take the ``<expression>`` values for each ``<formula>`` value and 
+  flatten them into a list. A single aggregation function is applied to all the values.
+- Monotonic aggregates take an ``<expression>`` for each value given by the ``<formula>``, 
+  and create combinations of all the possible values. The aggregation 
+  function is applied to each of the resulting combinations.
+
+In general, if the ``<expression>`` is total and functional, then monotonic aggregates are 
+equivalent to standard aggregates. Results differ when there is not precisely one ``<expression>`` 
+value for each value generated by the ``<formula>``:
+
+- If there are missing ``<expression>`` values (that is, there is less than one 
+  ``<expression>`` value for each value generated by the ``<formula>``), monotonic aggregates 
```

Comment:
I think this should be something like "for a value generated by the ``<formula>``" otherwise it reads to me like every value must be missing in this case.

"there is less than one  ``<expression>``" might be simpler as "there is no ``<expression>``" as the numbers here are natural numbers.